### PR TITLE
Fix/issue#524 crud

### PR DESCRIPTION
**Pull request by ibelar**
Fix issue #524 - Keep sort and pagination value when editing field,

### CODE REVIEW

File: src/CRUD.php [hunk 1]
```diff
@@ -165,16 +165,7 @@ public function initCreate()
...
     {
-        return [
-            // reload Grid
-            (new jQuery($this))->trigger('reload'),
-
-            // close modal
-            new jsExpression('$(".atk-dialog-content").trigger("close")'),
-
-            // display notification
-            $this->factory($this->notifyCreate ?: $this->notifyDefault),
```
Comment (DarkSide666) [inline src/CRUD.php]: You are loosing notifyCreate message now.

File: src/CRUD.php [hunk 2]
```diff
@@ -68,7 +68,7 @@ public function init()
     {
         parent::init();
 
-        $this->on('reload', $this->jsReload());
+        $this->stickyGet($this->paginator->name);
```
Comment (DarkSide666) [inline src/CRUD.php]: probably same issue is with QuickSearch / Filter

File: src/CRUD.php (hunk 1 above)
Comment (ibelar) [reply src/CRUD.php]: should be ok now!

File: src/CRUD.php (hunk 2 above)
Comment (ibelar) [reply src/CRUD.php]: yes!

**Pull request merged**